frappe/printing/page/print_format_builder/print_format_builder.js

```diff
@@ -497,10 +502,12 @@ frappe.PrintFormatBuilder = class PrintFormatBuilder {
 			});
 
 			d.set_value("label", field.attr("data-label"));
+			d.set_value("nolabel", +field.attr("data-nolabel"));
```

Comment:
What's the reason for the `+`, are there any cases where we have some invalid data here that'd need to get converted?

Reply:
You're right, it's not necessary, because _ControlCheck.validate_ returns `cint(value)` which works fine for "0" and "1".

---

The only place where it's required is here (where the `+` could be replaced with `cint`):

https://github.com/frappe/frappe/blob/1210ab33dc059560fce70b540471e33e1e520c74/frappe/printing/page/print_format_builder/print_format_builder.js#L817-L819